Integrations/EDL/EDL.py

```diff
@@ -151,6 +167,16 @@ def route_edl_values() -> Response:
     Main handler for values saved in the integration context
     """
     params = demisto.params()
+
+    username: str = params.get('credentials', {}).get('identifier')
+    password: str = params.get('credentials', {}).get('password')
+    if username and password:
+        headers: dict = cast(Dict[Any, Any], request.headers)
```

Comment:
i assume you did the cast cause of mypy?
what if we do some thing like `headers: Dict[Any, Any] =  request.headers` ?